.github/workflows/merge.yml

```diff
@@ -16,10 +16,15 @@ jobs:
         uses: ./.github/actions/setup
       - name: Generate TypeDoc documentation
         uses: ./.github/actions/build-docs
-      - name: Deploy documentation to gh-pages
-        uses: s0/git-publish-subdir-action@develop
+      - name: Push documentation artifacts to sdk-docs
+        uses: cpina/github-action-push-to-another-repository@main
         env:
-          REPO: self
-          BRANCH: gh-pages
-          FOLDER: docs
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SSH_DEPLOY_KEY: ${{ secrets.SSH_DEPLOY_KEY }}
+        with:
+          source-directory: docs
+          destination-github-username: pinecone-io
+          destination-repository-name: sdk-docs
+          user-email: clients@pinecone.io
+          target-branch: main
+          target-directory: typescript
+          commit-message: 'TypeScript: automated documentation build'
```

Comment:
A git SHA or something in this title might be nice so we can correlate the changes later, if needed.

Reply:
Good idea, will get that added.